cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/RegulatingControlMappingForGenerators.java

```diff
@@ -96,7 +96,7 @@ private boolean setRegulatingControlVoltage(String controlId,
         if (control.targetValue <= 0.0 || Double.isNaN(control.targetValue)) {
             targetV = terminal.getVoltageLevel().getNominalV();
             terminal = gen.getTerminal();
-            context.fixed(controlId, "Invalid value for regulating target value. Regulation considered as local.", control.targetValue, targetV);
+            context.fixed(controlId, () -> "Invalid value for regulating target value. Regulation considered as local.", control.targetValue, targetV);
```

Comment:
Introduce a version without supplier: there is no reason to create a supplier here, a simple String is OK as there is no parameters